server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateCleanupManager.java

```diff
@@ -344,6 +360,14 @@ void deleteStalePaths(String clusterName, String clusterUUID, List<String> stale
         );
     }
 
+    void deleteStaleIndexRoutingPaths(List<String> stalePaths) throws IOException {
+        logger.debug(() -> String.format(Locale.ROOT, "Deleting stale index routing files from remote - %s", stalePaths));
```

Comment:
Please avoid String format, logger has implicit formatting options use that instead

Reply:
Removed String format.